recipes/libev/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libev" %}
+{% set version = "4.24" %}
+{% set sha256 = "973593d3479abdf657674a55afe5f78624b0e440614e2b8cb3a07f16d4d7f821" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://dist.schmorp.de/libev/Attic/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
```

Comment:
Well I suppose it doesn't *hurt* to include any of these, even if they're not needed. We can iterate on this in the feedstock after it is created.